### PR TITLE
Add new asset image uploading animation

### PR DESCRIPTION
**Pull request by SaraVieira**
## Before requesting a review

- [ ] if the PR is WIP - use draft mode
- [x] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")
- [ ] what kind of review is needed?
  - [x] conceptual
  - [ ] detailed
  - [ ] with testing

## Test cases

- [ ] step by step interaction description and what is expected to happen

## Before merging

- [x] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-designer/blob/main/apps/designer/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `designer/env-check.js` if mandatory

CleanShot 2022-08-09 at 20 41 24

### CODE REVIEW

File: apps/designer/app/designer/features/sidebar-left/panels/components/uploading-animation.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  isUploading,
+  isDeleting,
+}: {
+  isUploading: boolean;
+  isDeleting: boolean;
+}) => {
+  const progressBarPercentage = useFakeProgress({ isUploading, isDeleting });
+  return (
+    <svg
+      style={{ position: "relative", zIndex: 10 }}
```
Comment (kof) [inline apps/designer/app/designer/features/sidebar-left/panels/components/uploading-animation.tsx:36]: lol, why not 999?

Comment (SaraVieira) [reply apps/designer/app/designer/features/sidebar-left/panels/components/uploading-animation.tsx:36]: I am an adult, i use zIndex carefully ahaha

File: apps/designer/app/designer/features/sidebar-left/panels/components/uploading-animation.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        percentage < 100 ? percentage + 1 : percentage
+      );
+    },
+    isUploading || isDeleting ? 100 : null
+  );
+
+  return progressBarPercentage;
+};
+
+export const UploadingAnimation = ({
```
Comment (kof) [inline apps/designer/app/designer/features/sidebar-left/panels/components/uploading-animation.tsx:26]: Generally there is quite a bit of accessibility stuff baked in in radix's progress bar

https://github.com/radix-ui/primitives/blob/main/packages/react/progress/src/Progress.tsx

File: apps/designer/app/designer/features/sidebar-left/panels/components/uploading-animation.tsx (hunk 1 above)
Comment (kof) [reply apps/designer/app/designer/features/sidebar-left/panels/components/uploading-animation.tsx:36]: ok it seems youcan use Radix's Progress and pass asChild prop and render the SVG inside, something like

```
<ProgressIndicator asChild>
   <CircularProgress />
</ProgressIndicator>
```
Circular Progress will receive all the props from radix

Comment (kof) [reply apps/designer/app/designer/features/sidebar-left/panels/components/uploading-animation.tsx:36]: > I am an adult, i use zIndex carefully ahaha

we have a few predefined levels in the design system, you should be ok with the first one

**Pull request merged**